internal/publish/database/exposure_test.go

```diff
@@ -756,7 +756,11 @@ func TestIterateExposuresCursor(t *testing.T) {
 	if err == nil {
 		t.Fatal("expected error")
 	}
-	if diff := cmp.Diff(exposures[:2], seen, ignoreUnexportedExposure); diff != "" {
+	// IterateExposures doesn't guarantee the order, sort to make this test deterministic
+	sortExposureFunc := func(i, j int) bool {
```

Comment:
I think this needs to take in two `*model.Exposure` params and return which true if the first is "less than" the other

Reply:
That's right, fixed.

Surprisingly the previous one worked...